lang/en/theme_boost_union.php

```diff
@@ -724,6 +724,13 @@
 $string['tilebackgroundimagesetting_desc'] = 'Here, you can upload an image file which will be shown as background image behind the content of the advertisement tile {$a->no}. Please make sure or check that the content is still readable on the background image. This is an optional setting, the advertisement tile will work even if you do not upload any background image.';
 $string['tilecontentsetting'] = 'Advertisement tile {$a->no} content';
 $string['tilecontentsetting_desc'] = 'Here, you enter the content which should be displayed in the advertisement tile {$a->no}. The content is displayed in the middle of the tile. This is an optional setting, the advertisement tile will be shown even if you do not set any content.';
+$string['tilecontentstylesetting'] = 'Advertisement tile {$a->no} content style';
+$string['tilecontentstylesetting_dark'] = 'Dark (Black font color for dark background images)';
+$string['tilecontentstylesetting_darkshadow'] = 'Dark & Shadow (Black font color with a light shadow for light background images)';
```

Comment:
Dark & Shadow (black font color with a light shadow for light background images)

Begin lower case inside the brackets.